Sources/ForkedModelMacros/ForkedModelMacro.swift

```diff
@@ -59,16 +59,17 @@ public struct ForkedModelMacro: ExtensionMacro {
         guard structDecl.allStoredPropertiesHaveDefaultValue else {
             throw ForkedModelError.nonOptionalStoredPropertiesMustHaveDefaultValues
         }
-        
+
         // Get all vars
         let allPropertyVars: [VariableDeclSyntax] = structDecl.memberBlock.members.compactMap { member -> VariableDeclSyntax? in
             guard let varSyntax = member.decl.as(VariableDeclSyntax.self) else { return nil }
+            guard !varSyntax.isComputedVar() else { return nil }
             return varSyntax
         }
         
         // Gather names of all mergeable properties
         let mergePropertyVars: [MergePropertyVar] = try structDecl.memberBlock.members.compactMap { member -> MergePropertyVar? in
-            guard let varSyntax = member.decl.as(VariableDeclSyntax.self), varSyntax.isMerged()
+            guard let varSyntax = member.decl.as(VariableDeclSyntax.self), varSyntax.isMerged(), !varSyntax.isComputedVar()
```

Comment:
Maybe rename `isComputedVar` to `isComputed`, to match `isMerged` naming scheme.